jaide/cli.py

```diff
@@ -298,8 +324,10 @@ def main(ctx, host, password, port, quiet, session_timeout, connect_timeout,
               "--at and --confirm are mutually exclusive, and confirm will"
               " override. Can be in one of two formats: hh:mm[:ss]  or  "
               "yyyy-mm-dd hh:mm[:ss]")
+@click.option('-A', '--action', 'action', default='set', callback=action_validate, help="Specify the method to load the configuration: "
+              "`set`, `merge`, `override`, `replace`, `update`")
 @click.pass_context
-def commit(ctx, commands, blank, check, sync, comment, confirm, at_time):
+def commit(ctx, commands, blank, check, sync, comment, confirm, at_time, action):
```

Comment:
80 chars?

There are a ton more of these throughout, but I'm going to stop mentioning them.